scripts/auto_pr_on_failure.sh

```
@@ -0,0 +1,151 @@
+	LATEST_SUMMARY=$(ls -1t ${SUMMARY_FILE} | head -n1)
+	echo "Included summary: ${LATEST_SUMMARY}"
+	echo
+	awk 'NR<=200 {print}' "${LATEST_SUMMARY}" || true
+else
+	echo "No repair summary found."
+fi
+echo
+echo "## Compiler Error Snippet (tail)"
+echo "```"
+tail -n 200 "${LATEST_LOG}" || true
+echo "```"
```

Shellcheck (0.11.0)

[warning] This is interpreted as a command name ending with space. Double check syntax.

(SC2288)